### ISSUE TITLE
What introductory/documentation improvements should we make?

### ISSUE THREAD

**Issue by rnorth** [testcontainers/testcontainers-java#31]
Basically, does the current README adequately explain this project, or does it need to be fairly radically updated? I'd value your feedback on how to make it easier to grok what this is all about :smile:

**Comment by rnorth** [comment]
paging @swissarmykirpan @outofcoffee @haines @optilude...

**Comment by swissarmykirpan** [comment]
I think the current README covers a fair bit around HOW to make use of it, but not really WHERE to make use of it. i.e.as part of a development workflow (for which we will probably need a website).

**Comment by rnorth** [comment]
> I think the current README covers a fair bit around HOW to make use of it, but not really WHERE to make use of it. i.e.as part of a development workflow (for which we will probably need a website).

I've updated the README today - please have a look and see. Hopefully this explains a bit better, though there's perhaps still some way to go (detailed steps should go on the wiki).

**Comment by rnorth** [comment]
I've done a whole raft of docs improvements, now visible here: http://testcontainers.viewdocs.io/testcontainers-java/

**Comment by rnorth** [comment]
Will close for now - anything that looks too vague in the future, let's open new tickets.

**Comment by kaiwinter** [comment]
Somewhere should be mentioned that Java 8 is required. I spent some time figuring out why the `ContainerDatabaseDriver` of oracle-xe:0.9.8 doesn't register while the one of oracle-xe:0.9.7 does. I got no exceptions the class just wasn´t loaded. 
`Class.forName("org.testcontainers.jdbc.ContainerDatabaseDriver");` finally explains it: `java.lang.UnsupportedClassVersionError: org/testcontainers/jdbc/ContainerDatabaseDriver : Unsupported major.minor version 52.0`
